safekeeper/src/bin/safekeeper.rs

```diff
@@ -344,8 +347,16 @@ async fn main() -> anyhow::Result<()> {
             Some(SecretString::from(v))
         }
         Err(VarError::NotPresent) => {
```

Comment:
worth adding a comment that we first try env var and then path, and that the former can be dropped after we migrate to path in the field.

Reply:
added